prod-docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+  image-pegasus-cim-mcgill-ca:
+    image: nginx:latest
+    restart: unless-stopped
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.atp-pegasus.rule=Host(`pegasus.cim.mcgill.ca`) && PathPrefix(`/image`)"
+      - "traefik.http.routers.atp-pegasus.tls.certresolver=myresolver"
+      - "traefik.http.middlewares.peg-redirect.redirectregex.regex=^(https?)://pegasus.cim.mcgill.ca/image(/?.*)"
+      - "traefik.http.middlewares.peg-redirect.redirectregex.replacement=$${1}://image.a11y.mcgill.ca$${2}"
+      - "traefik.http.middlewares.peg-redirect.redirectregex.permanent=true"
+      - "traefik.http.routers.atp-pegasus.middlewares=peg-redirect@docker"
+
+      - "traefik.http.routers.image.rule=Host(`image.a11y.mcgill.ca`) || Host(`image.ally.mcgill.ca`) || Host(`image.accessibility.mcgill.ca`)"
+      - "traefik.http.middlewares.ally-redirect.redirectregex.regex=^(https?)://image.(ally|accessibility).mcgill.ca(/?.*)"
+      - "traefik.http.middlewares.ally-redirect.redirectregex.replacement=$${1}://image.a11y.mcgill.ca$${3}"
+      - "traefik.http.middlewares.ally-redirect.redirectregex.permanent=true"
+      - "traefik.http.routers.image.middlewares=ally-redirect@docker"
+      - "traefik.http.routers.image.tls.certresolver=myresolver"
+      - traefik.docker.network=traefik
+    environment:
+      - SERVER_NAME=image.a11y.mcgill.ca
+    volumes:
+      - ./www/html:/var/www/html:ro
+      - ./www/image.conf:/etc/nginx/templates/image.conf.template:ro
+      - website-logs:/var/log/nginx
+      - ./www/passwords:/etc/nginx/passwords
+      - ./resources/current:/var/www/resources:ro
```

Comment:
Maybe include a note that these local mounts refer to files used to manage our website, and are unrelated to the "IMAGE server" stuff here. I can see someone getting confused by this.

Reply:
I originally thought we should potentially pull everything to do with the website out of the core docker-compose and put it in another directory with its own docker-compose, but the web front end is kind of intermixed with the endpoints for server requests. Is there a clean way to separate these, that would help with understanding? If unclear, I think documentation in place is the best approach, and people can just remove the tagged lines for the website itself if they only want to serve requests.